run.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
At the hackathon we chose the BSD license for code related to this project

Reply:
Should probably just delete this section and reference the license in `LICENSE`, case of copy-pasta with old files.